solr/modules/jwt-auth/src/java/org/apache/solr/security/jwt/JWTIssuerConfig.java

```diff
@@ -315,6 +327,24 @@ public JWTIssuerConfig setAuthorizationEndpoint(String authorizationEndpoint) {
     return this;
   }
 
+  public String getTokenEndpoint() {
+    return tokenEndpoint;
+  }
+
+  public JWTIssuerConfig setTokenEndpoint(String tokenEndpoint) {
+    this.tokenEndpoint = tokenEndpoint;
+    return this;
+  }
+
+  public String getAuthorizationFlow() {
+    return authorizationFlow;
+  }
+
+  public JWTIssuerConfig setAuthorizationFlow(String authorizationFlow) {
+    this.authorizationFlow = authorizationFlow;
```

Comment:
Should we not validate the flow value early and fail if not either of the two allowed values? I'm thinking we can also let this default to `implicit` here in the backend instead of in the frontend?

Reply:
I've added a validation test to _setAuthorizationFlow_. The reason I am not doing it earlier is:
- I am following the same logic/style than the other methods (like _setJwksUrl_)
- Adding it earlier would lead to too much verbose code for such basic test.

I have also set _implict_ as default. Will change the front code too.